host/4/publish.yml

```diff
@@ -295,36 +295,6 @@ steps:
     displayName: tag and push node18 images
     continueOnError: false
 
-  - bash: |
-      set -e
-      docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7
-      docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-slim
-      docker pull $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice
-
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7 $TARGET_REGISTRY/powershell:$(TargetVersion)
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-slim $TARGET_REGISTRY/powershell:$(TargetVersion)-slim
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice-quickstart
-
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7 $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-slim $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-slim
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice
-      docker tag $SOURCE_REGISTRY/powershell:$(PrivateVersion)-powershell7-appservice $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice-quickstart
-
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-slim
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice-quickstart
-
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-slim
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice-quickstart
```

Comment:
Would you like me to delete the existing powershell 7 images? Or keep them up to support customers who are still on them ?

Reply:
unfortunately we cannot remove as the images are public but we can stop building them as it is not a supported version